.appveyor.yml

```diff
@@ -8,8 +8,9 @@ skip_tags: false
 
 install:
   - ps: |
-      $latest = (Invoke-WebRequest "http://downloads.dlang.org/releases/LATEST").toString();
-      $url = "http://downloads.dlang.org/releases/2.x/$($latest)/dmd.$($latest).windows.7z";
+      #$latest = (Invoke-WebRequest "http://downloads.dlang.org/releases/LATEST").toString();
+      #$url = "http://downloads.dlang.org/releases/2.x/$($latest)/dmd.$($latest).windows.7z";
+      $url = "http://downloads.dlang.org/releases/2.x/2.088.1/dmd.2.088.1.windows.7z"
```

Comment:
Not sure you want this...

Reply:
current io code can build up to dmdfe 2.088.1

Reply:
Yeah, we can update this when the bugs are fixed in the latest compiler.